docs/README.md

```diff
@@ -74,6 +74,26 @@ If you want to set up everything manually, you can checkout the [Feathr CLI depl
 | [Fraud Detection Demo](./samples/fraud_detection_demo.ipynb)                        | An example to demonstrate Feature Store using multiple data sources such as user account and transaction data.                        | Azure Synapse, Databricks, Local Spark |
 | [Product Recommendation Demo](./samples/product_recommendation_demo_advanced.ipynb) | Feathr Feature Store example notebook with a product recommendation scenario                                                          | Azure Synapse, Databricks, Local Spark |
 
+## 🏃 Getting Started with Feathr
+
+The best way to try out Feathr is to use the [Feathr Sandbox](https://feathr-ai.github.io/feathr/quickstart_local_sandbox.html) which is a self-contained container with most of Feathr's capabilities. To use it, simply run this command:
```

Comment:
`The best way` seems to be a risky statement. Shall we confirm it with other team members?

Reply:
changed to `easiest`